packages/webapp/src/components/Animals/AddAnimalsFormCard/AddAnimalsFormCard.tsx

```diff
@@ -83,6 +87,14 @@ export default function AddAnimalsFormCard({
     }
   }, []);
 
+  useEffect(() => {
+    if (shouldCreateIndividualProfiles && watchAnimalCount > ANIMAL_COUNT_LIMIT) {
+      trigger(`${namePrefix}${BasicsFields.COUNT}`);
+    } else {
+      clearErrors(`${namePrefix}${BasicsFields.COUNT}`);
+    }
```

Comment:
The error will be cleared automatically by `trigger`, so `trigger(`${namePrefix}${BasicsFields.COUNT}`);` without any conditions should work.

It would be even better if this could be handled in the `onChange` for the checkbox. Have you tried it?  I found it was possible by adding a setTimeout like this:
```
onIndividualProfilesCheck?.((e.target as HTMLInputElement).checked);
setTimeout(() => {
  trigger(`${namePrefix}${BasicsFields.COUNT}`);
}, 0);
```